test/link/table.txt

```diff
@@ -44,11 +44,11 @@ Elem:
 Custom:
  - name: "reloc.Elem"
   - section: Elem
-   - R_FUNC_INDEX_LEB   idx=0    addend=0    offset=0x6(file=0x38)
```

Comment:
Sorry I mean MEMORY_ADDR relocs.  They are the only type with an addend ( https://github.com/WebAssembly/tool-conventions/blob/master/Linking.md)